src/types/Errors.ts

```diff
@@ -0,0 +1,8 @@
+export enum Errors {
+  Load = 'Unable to load todos',
+  Empty = 'Title should not be empty',
+  Add = 'Unable to add a todo',
+  Delete = 'Unable to delete a todo',
+  Update = 'Unable to update a todo',
+  NoError = '',
```

Comment:
Avoid using empty strings for error handling. It is better to use null to show that there is no error. An empty string may cause confusion or unexpected behavior in some cases.